src/test/interface/io/json/json_data_test.cpp

```diff
@@ -52,6 +54,27 @@ void test_real_var(cmdstan::json::json_data &jdata, const std::string &text,
     EXPECT_EQ(expected_vals[i], vals[i]);
 }
 
+void test_complex_var(cmdstan::json::json_data &jdata, const std::string &text,
+                      const std::string &name,
+                      const std::vector<std::complex<double>> &expected_vals,
+                      const std::vector<size_t> &expected_dims) {
+  EXPECT_EQ(true, (jdata.contains_r(name) || jdata.contains_i(name)));
+  std::vector<size_t> dims = jdata.dims_r(name);
+  if (dims.size() == 1) {
+    dims.clear();
+  } else {
+    dims.pop_back();
+  }
+  EXPECT_EQ(expected_dims.size(), dims.size());
+  for (size_t i = 0; i < dims.size(); i++)
+    EXPECT_EQ(expected_dims[i], dims[i]);
+  std::vector<std::complex<double>> vals = jdata.vals_c(name);
+  ;
```

Comment:
Remove the stray semicolon.  It's a valid empty statement, but it's not doing anything.